spec

```diff
@@ -6807,13 +6807,15 @@ endseg
 beginseg
     name "elf_message_field"
     romalign 0x1000
-    include "build/baserom/elf_message_field.o"
+    include "build/src/elf_message/elf_message_field.o"
+    number 0
```

Comment:
Do we need to define these segments as having number 0? I thought I remembered these were never accessed in memory, in which case they probably don't need a number at all?

Reply:
We don't need to, however I believe it is correct to do so, otherwise they'll be put at some weird address based on whatever was set prior in the spec, which imo is strictly worse than just declaring a number, or even an address, even if the address isn't currently relevant to anything in the codebase.

Reply:
I can see that yeah. In that case it's probably fine, tho I'm worried about the noise it would add if we applied it everywhere, and how it would look like with defines for all numbers, but we can probably figure it out later